### PR TITLE
V3.2 FORM - Dynamic Text Substitution - UNIQUE Sticker Unown form

### PR DESCRIPTION
**Pull request by Sttug**
## Description
Add Support to send Dynamic text
to send form info
Add send stickers accord to Unown form in Telegram

## How Has This Been Tested?
tested in my own server send fake notification to webhook
using python 

https://pastebin.com/pN7ynqj5

## Screenshots (if appropriate):
normal  notification whit form 
http://imgur.com/a/IsczI
normal  notification whiteout  form
http://imgur.com/a/F4OIh
normal  notification other Pokemon whiteout  form
http://imgur.com/a/ROIiS

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X ] New feature (non-breaking change which adds functionality)
- [ X ] Breaking change (fix or feature that would cause existing functionality to change)

## Wiki Update
Need modify dynamic text to add  `<form>` tag

## Checklist:

- [ X ] My code follows the code style of this project.
- [ X ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by kbtbc** [comment]
I suggest than when <form> is not present, that this value defaults to '' instead of 'unkn'  ... since currently it is only for unowns, that could avoid filling notifications w/ unnecessary text.

File: PokeAlarm/WebhookStructs.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def pokemon(data):
             'lat': float(data['latitude']),
             'lng': float(data['longitude']),
             'cp': check_for_none(int, data.get('cp'), '?'),
-            'level': check_for_none(int, data.get('pokemon_level'), '?'),
+            'level': 2,
```
Comment (myntath) [inline PokeAlarm/WebhookStructs.py]: Any reason for this? it would seem this would break level reporting.

Comment (Sttug) [inline PokeAlarm/WebhookStructs.py]: pr https://github.com/RocketMap/RocketMap/pull/1991 dont commit , if leave black active ignore missing.

**Comment by Sttug** [comment]
i have right @myntath  i make other pr but i still wait for real one appear xD  https://github.com/Sttug/PokeAlarm/tree/v3.2
 if u like test !

**Pull request closed without merging**